index.html

```diff
@@ -1770,12 +1770,8 @@ <h4>
           <dt>arrayBuffer</dt>
           <ol>
             <li>
-              If the |recordType| value is equal to "`json`" or "`opaque`", then
-              return an {{ArrayBuffer}} whose contents are the
-              |bytes|. Re-[= exception/throw =] any exceptions.
-            </li>
-            <li>
-              Otherwise, return `null`.
+              Return an {{ArrayBuffer}} whose contents are the |bytes|.
+              Re-[= exception/throw =] any exceptions.
```

Comment:
Can this actually throw?

Reply:
Do you mean "Throw any exceptions" instead of "Re-throw any exceptions"?

Reply:
In https://w3c.github.io/push-api/#widl-PushMessageData-arrayBuffer-ArrayBuffer, it says:
> The arrayBuffer() method, when invoked, MUST return an ArrayBuffer whose contents are bytes. Exceptions thrown during the creation of the ArrayBuffer object are re-thrown.